src/microsim/transportables/MSPModel_JuPedSim.h

```diff
@@ -114,7 +121,10 @@ class MSPModel_JuPedSim : public MSPModel {
         Position myDestination;
         JPS_AgentId myAgentId;
         Position myPosition;
+        Position myPreviousPosition; // Will be initialized to zero automatically.
         double myAngle;
+        double myLanePosition;
+        MSEdge* myCurrentEdge;
```

Comment:
This seems to be unused

Reply:
`myCurrentEdge` has been removed, thanks.